moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner/chomp_parameters.h

```diff
@@ -250,6 +254,16 @@ inline std::string ChompParameters::getTrajectoryInitializationMethod() const
   return trajectory_initialization_method_;
 }
 
+inline bool ChompParameters::getEnableFailureRecovery() const
```

Comment:
in most all cases you do not need ``inline``as the compiler is very smart these days.

Reply:
removed `inline`